cips/cip-3.md

```diff
@@ -0,0 +1,66 @@
+---
+cip: 3
+title: Diamonds Paid In CLOUT
+author: RedPartyHat <@redpartyhat>
+discussions-url: https://github.com/bitclout/cips/discussions/26
+created: 2021-08-04
+---
+
+## One-Line Description
+
+Pay for diamonds with CLOUT instead of creator coins.
+
+## Explanation and Motivation
+
+"Diamonds" are a tool that allow users to show their love for a creator's work on BitClout. A user
+can give up to six diamonds per post and each diamond given comes with a reward for the creator. A
+single diamond gives the creator ~0.005 USD (half a penny) at the time of writing. Each additional
+diamond 10x's the reward, up to ~$500 USD for six diamonds. This has already become a significant
+source of income for many creators.
+
+Currently, the reward given with each diamond is paid in the giver's creator coin. This is great
+for creating a dense creator coin ownership graph. Diamond recipient's often do not sell the creator
+coins that they receive, which gives them a growing financial stake in the success of all of their
+supporters. However, this is not great for compensating creators and lines diamond givers up for a
+dump event if creators need to cash out for raw CLOUT in order to support themselves.
+
+Making diamonds paid in CLOUT instead of creator coins will maximize the financial rewards for
+creators and simplify the platform for newcomers.
+
+## Specification
+
+Implementation will be similar to any forking change. A blockheight will be used to give nodes all
+nodes in the network time to upgrade. After that blockheight, all diamond transactions will be paid
+with CLOUT instead of creator coin.
+
+## Data Storage Change
+
+This change will increase the storage requirements for diamonds on the chain. Creator coins are
+stored using a balance model on the chain. Therefore, in the existing state, if a diamond receiver
+already holds the creator coin of the diamond giver the database will only be updated. In contrast,
+CLOUT is paid out using UTXOs. Each new diamond transaction will create an entry in the database
+everytime a user is given a diamond. This will be a manageable and temporary increase in storage
+that will be relieved when CLOUT is moved from UTXOs to a balance model.
+
+## Backwards Compatibility
+
+Diamonds given before the fork will not be compatible with the consensus logic after the fork and
+vice versa.
+
+## Security Considerations
+
+This upgrade should not significantly impact the security of the chain. Typical care should be taken
+in reviewing the code to ensure that there are no "money printer" bugs in the revised transaction.
+
+## Alternate designs considered
+
+An alternative design would be to provide users with the choice to pay for diamonds in either CLOUT
+or creator coin. A sensible default could be chosen and the user's preference could be stored on
+their profile entry. This option is attractive in that it maximizes choice for users but it adds
```

Comment:
I think thought should be given to other potential negative impacts of removing creator coin as the payment mechanism.

The big risk for me is a possible reduction in diamonds given.

Right now it is in the interest of a creator to give diamonds because it reduces impact of people selling, increases people holding their coin, and reduces high coin holder concentration / HHI - key metric used by proper clout and `@artz` for determining risk of investing in creator coin.

Reply:
Giving users the ability to choose will ultimately make it easier in the long run.

I second what @tijno stated:
The current micro-economy created around CC-diamonds, might falter as most people own their coin (initial buy-in of their own coin that got pumped exponentially on the bonding curve), yet have a balance of 0 clout.

These accounts may become tempted to sell their own coin holdings, in order to tip with diamonds.

This worst case scenario could cause mass-paranoia on the platform, as we had seen with the scam-era (June saw far less investments in smaller coins due to fear from getting rug pulled).

If, however, a user defined choice is given, this threat is eliminated.

Reply:
In my opinion, the original intention of diamonds was not to create a mechanism to diversify the risk of a creator's coin being sold but rather as a way to incentivise quality content creation to monetise it as an opposition to the current situation on other social media networks. Furthermore, some Bitclout users are questioning whether the mechanism to sell 'diamonds dust' is efficient and are requesting a mass/bulk option to sell them to have access to liquidity; this goes against the concept of using the diamond feature as a diversification tool IMHO.

Regarding the fact that people don't have enough $CLOUT in their wallets is something I would like to have more data from the current blockchain (unfortunately, I don't know how to retrieve this data). However, If that would be the case, the NFT ecosystem wouldn't be sustainable, correct? As you need to bid and ultimately buy the NFT with $CLOUT and not with creator coins. Therefore, I would be more inclined to think that is not going to be a problem.

Finally, the proposal aligned to the Bitclout [Vision](https://docs.bitclout.com/the-vision) documentation and this specific phrase:

"There has been relatively little innovation with regard to how creators monetise. Most creators earn much less than they should on existing platforms, if they earn anything at all."

As I mentioned in the CIP Discussion [#26](https://github.com/bitclout/cips/discussions/26), the primary goal of this change is to facilitate access to liquidity for the content creator and improve the internal economy of Bitclout since access to this liquidity will benefit the network to increase the buying of other creators coins, invest on NFTs or invest in your own creator coin.

Reply:
I dont think this addresses the concerns raised in the above cips:

* how to achieve dense distribution of creator coin if diamonds can no longer be used for this

* how to mitigate the risk that less diamonds will be given.

The change is clearly going ahead without consideration of the above points.